Tests/Unit/Logger/LoRaConsoleLoggerTests.cs

```diff
@@ -69,6 +69,51 @@ public void When_Message_Already_Prefixed_Scope_Should_Not_Be_Added()
             moqInfo.Verify(x => x.Invoke(It.Is<string>(x => x == message)), Times.Once);
         }
 
+        [Theory]
+        [InlineData("EUI", null, null, "EUI foo")]
+        [InlineData(null, "ADDR", null, "ADDR foo")]
+        [InlineData(null, null, 1, "0000000000000001 foo")]
+        [InlineData("EUI", null, 1, "EUI foo")]
+        [InlineData("EUI", "ADDR", null, "EUI foo")]
+        [InlineData(null, "ADDR", 1, "ADDR foo")]
+        public void When_DevEUI_And_DevAddr_Scope_Set_DevEUI_Scope_Is_Set(string devEuiScope, string devAddrScope, int? stationEuiScope, string expected)
```

Comment:
[nit]Test name could be improved e.g. `When_Multiple_Scopes_Set_DevEUI_Preferred_Over_DevAddr_Preferred_Over_StationEUI`